xml/System.Text.Json/Utf8JsonWriter.xml

```diff
@@ -419,10 +497,14 @@
         <Parameter Name="escape" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="propertyName">To be added.</param>
-        <param name="escape">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="propertyName">The UTF-16 encoded property name of the JSON object to be transcoded and written as UTF-8.</param>
```

Comment:
```suggestion
        <param name="propertyName">The UTF-16 encoded property name of the JSON object to transcode and write as UTF-8.</param>
```
This change can also be made to a number of the following `propertyName` descriptions.